src/synopsys-action/service/impl/enterprise/v1/github-client-service-v1.ts

```diff
@@ -0,0 +1,3 @@
+import {GithubClientServiceBase} from '../../github-client-service-base'
+
+export class GithubClientServiceV1 extends GithubClientServiceBase {}
```

Comment:
We should proper comment for this class mentioning, why it is left empty

Reply:
Added comments